changelog

```diff
@@ -214,6 +214,9 @@
 	generation time if there are any orphan OpenACC or OpenMP
 	directives e.g. OpenMP loop should be within OpenMP parallel.
 
+        69) PR #970 for #955. Use Jinja to generate the netcdf extraction
```

Comment:
Not sure what this is doing in this PR - perhaps git has got confused over a merge?

Reply:
I believe that is actually correct - it is the merge of introducing jinja extraction, which at that stage is only dl_esm_inf. This PR then only adds LFRic.